torch/_inductor/simple_fsdp.py

```diff
@@ -122,7 +117,7 @@ def reorder_reduce_scatter(
 
 
 def get_node_type(node, prev_nodes=None) -> int:
-    # node_type: {0: all gather; 1: wait_tensor; 2: computation; 3: reduce scatter; 4: convert_element_type after rs}
+    # node_type: {0: all gather; 1: wait_tensor; 2: computation; 3: reduce scatter;}
     node_type = NodeType.COMPUTE
     if not isinstance(node, scheduler.FusedSchedulerNode):
         if (
```

Comment:
Let's use `if-else` to refactor the code in this function. Otherwise every `if` needs to be run.